python/graphframes/tests.py

```diff
@@ -333,8 +333,6 @@ def test_page_rank(self):
         self._hasCols(pr, vcols=['id', 'pagerank'], ecols=['src', 'dst', 'weight'])
 
     def test_parallel_personalized_page_rank(self):
-        if not GraphFrameTestUtils.spark_at_least_of_version("2.1"):
```

Comment:
Do we want to remove all dependency of older version in the tests?
In `test_connected_components_friends`, we also check the version for 2.0.